addons/web_editor/static/src/js/editor/summernote.js

```diff
@@ -2298,8 +2298,10 @@ $.summernote.pluginEvents.applyFont = function (event, editor, layoutInfo, color
       $font = $(node);
       className = dom.orderClass(node);
       style = dom.orderStyle(node);
+      hasBefore = window.getComputedStyle(node, '::before').content !== 'none';
+      hasAfter = window.getComputedStyle(node, '::after').content !== 'none';
```

Comment:
I used that same code recently and I also had to check that the `content` property was not falsy. Can't remember why but could not hurt to add this extra check.

Reply:
@qsm-odoo Not sure what you mean, if the content property is the empty string, it's falsy and also indicative that there actually is a pseudo-element. If you mean that I should check that getComputedStyle returns something on which I can access properties to avoid `cannot read property 'content' of undefined`, that should be guaranteed unless you pass something that does not implement the Element interface (like a text node), but in that case getComputedStyle will throw, not fail silently. This is not a concern in this case because node is a `<font>` or `<span>` element, but if it was a concern I would need to check the node type before this or use a try-catch, but I don't think it's necessary in this case. Maybe you can find back the place where you had to do that so we can make sure?

Reply:
I was talking about `.content` which may return `null` or `undefined` on some browsers which would be equivalent to `none`. If you trust it's not possible ever anymore, then you can r+ :+1:

Reply:
Ah gotcha, I was confused because you said falsy and we have a lot of empty string contents that actually display stuff. I can add a check for null and undefined too though.

Reply:
Yes, falsy was not a great term :wink: